environment-setup.md

```diff
@@ -0,0 +1,152 @@
+# 💻 Setting up your development environment
+
+There are many guides to setting up your computer for development. Each has its own merits. At DataMade, we perform a mix of maintaining legacy projects and developing new projects, often in parallel. Our toolkit is therefore optimized for managing many, isolated versions of packages and getting up and running on new projects quickly.
+
+#### Contents
+
+- [Version control](#version-control)
+- [Text editor](#text-editor)
+- [Python](#python)
+- [Docker](#docker)
+- [Security](#security)
+- [Data](#data)
+- [Geospatial data](#geospatial-data)
+- [Static sites](#static-sites)
+
+For Linux users, most of these tools are available as aptitude packages. When possible, prefer binary packages to building from source. For Mac users, most of these tools are available as [Homebrew](http://brew.sh/) packages. Homebrew is by far the easiest way to manage packages on OS X.
+
+## Version control
+
+We use GitHub and Git to keep our work under version control. Note that we prefer the `git` CLI to the GitHub desktop GUI.
+
+#### Packages
+
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+## Text editor
+
+Most of us use Sublime as our text editor of choice. If you do, too, then you will need to make some modifications to your settings.
+
+In the nav, under "Sublime Text," you will find a "Preferences" fly open, where you will see "Settings - User." Here, you can override the default settings (i.e., do not make changes to the "Settings - Default" file). In the User file, add two things:
+
+```json
+    "trim_trailing_white_space_on_save": false,
+    "tab_size": 4,
+```
+
+‼ **Note:** You are welcome to explore other text editor options, e.g., Atom, Vim, etc.
+
+#### Packages
+
+* [Sublime Text 3](https://www.sublimetext.com/3)
+
+## Python
+
+At DataMade, you'll run most Python processes in containers. However, it's still useful to have a fresh install of Python on your machine to keep your system Python isolated (it's important!) and to use a later version of Python (if you're on a Mac, your system Python is probably version 2.7).
```

Comment:
What is the "(it's important!)" parenthetical doing for us here?

Reply:
Vaguely underscoring why you might want to keep your system's Python isolated.